compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/AnnotationDesugar.java

```diff
@@ -628,6 +631,38 @@ private BLangLambdaFunction defineAnnotations(BLangTypeDefinition typeDef, BLang
         return lambdaFunction;
     }
 
+    private BLangLambdaFunction defineTupleAnnotations(BLangTypeDefinition typeDef, BLangPackage pkgNode, SymbolEnv env,
```

Comment:
Isn't this method pretty much the same as `org.wso2.ballerinalang.compiler.desugar.AnnotationDesugar#defineAnnotations(org.wso2.ballerinalang.compiler.tree.BLangTypeDefinition, org.wso2.ballerinalang.compiler.tree.BLangPackage, org.wso2.ballerinalang.compiler.semantics.model.SymbolEnv, org.ballerinalang.model.elements.PackageID, org.wso2.ballerinalang.compiler.semantics.model.symbols.BSymbol)`? Can't we use one method by accepting the list of fields/members?